src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -207,6 +207,11 @@ constructor of the ``Helper`` class.
       // ...
    }
 
+Please refer to the official `Protractor style-guide
+<https://www.protractortest.org/#/style-guide>`__
+for a better insight on how to write and structure tests
```

Comment:
```suggestion
for a better insight on how to write and structure tests,
```

Reply:
Looks like it's not correct to add a comma here - https://www.grammarly.com/blog/as-well-as-comma/